src/vos/tests/evt_ctl.c

```diff
@@ -1682,6 +1683,220 @@ test_evt_various_data_size_internal(void **state)
 	}
 }
 
+static void
+test_evt_node_size_internal(void **state)
+{
+	struct test_arg		*arg = *state;
+	daos_handle_t		 toh;
+	int			order_size, rc;
+	bool			evt_created;
+
+	for (order_size = 0; order_size < ORDER_TEST_SIZE; order_size++) {
+		evt_created = false;
+		rc = evt_create(arg->ta_root, ts_feats, order_size,
+				arg->ta_uma, &ts_evt_desc_cbs, &toh);
+		if ((order_size >= 4) && (order_size <= 128)) {
```

Comment:
you could probably do something like this to avoid needing the evt_created variable
if (order_size < 4 || order_size > 128) {
     assert_int_not_equal(rc, 0);
    continue;
}

assert_int_equal(rc, 0);
rc = evt_destroy(toh);
assert_int_equal(rc, 0);


Can we use the actual macros rather than 4 and 128?   If we need to move them up to evt_priv.h, I think that's fine.

Reply:
Done. Thanks.